packages/terra-demographics-banner/src/_sharedObjects.jsx

```diff
@@ -1,23 +1,46 @@
 import classNames from 'classnames/bind';
 import React from 'react';
+import VisuallyHiddenText from 'terra-visually-hidden-text';
 import styles from './DemographicsBanner.module.scss';
 
 const cx = classNames.bind(styles);
 
 // eslint-disable-next-line react/prop-types
-const DemographicsBannerValue = ({ label, value }) => (
-  <span className={cx('value')}>
-    { label && <span className={cx('value-label')}>{`${label}:`}</span> }
-    <b>{value}</b>
-  </span>
-);
+const DemographicsBannerValue = ({ label, value, abbrTitle }) => {
+  let valueLabelContent;
+
+  if (abbrTitle) {
```

Comment:
Should we still be checking that the `label` prop is valid here as well?

Reply:
I mostly assumed that if abbrTitle is provided, that the label has to be since that's the full text descriptor of the abbrTitle, but I can be more strict with the prop checking

Reply:
hmmm 🤔 I don't have a strong opinion. I just saw we checked for it previously before rendering it

Reply:
I'll make it more strict. I think that's what is best for our components